### DISCUSSION TITLE
Can't expand when there is multiple filter applied?

### DISCUSSION THREAD

**Discussion by plantshopping** [pocketbase/pocketbase#5730] [Q&A]
```
// Expands on user
   const data = await pb.collection("user_skills").getList(1, 50, {
      filter: `role="teacher"`,
      //   expand: "user",
    });

// Expands on user
   const data = await pb.collection("user_skills").getList(1, 50, {
      filter: `skill.name ?~ "${searchTerm.toLowerCase()}"`,
      //   expand: "user",
    });

// Won't expand on user
   const data = await pb.collection("user_skills").getList(1, 50, {
      filter: `skill.name ?~ "${searchTerm.toLowerCase()}" && role="teacher"`,
      //   expand: "user",
    });
```

**Comment by ganigeorgiev** [comment]
`expand` shouldn't be affected by the used filters.

It depends only on the View API rule of the collection that is being expanded. Double check your users View API rule and make sure that what you are experiencing is not cause by client-side cache (this is common with Next.js 14 because the global `fetch` is overwritten to have cache enabled by default).

If you are still not able to identify the culprit, please report with what PocketBase version you are testing and export your collections configuration as json (_Admin UI > Settings > Export collections_).
